src/helpers/cognito.ts

```diff
@@ -0,0 +1,39 @@
+import { CognitoIdentityProviderClientConfig } from '@aws-sdk/client-cognito-identity-provider';
+import { z } from 'zod';
+import {
+  assertMatcherHelperInputValue,
+  HelperZodSchema,
+  RetryableMatcherHelper,
+} from './internal';
+
+/**
+ * Cognito User helper input
+ */
+export type CognitoUserInput = {
+  userPoolId: string;
+  username: string;
+  clientConfig?: CognitoIdentityProviderClientConfig;
+};
+
+/**
+ * Cognito User schema
+ */
+const cognitoUserInputSchema: HelperZodSchema<typeof cognitoUser> = z.object({
+  userPoolId: z.string(),
+  username: z.string(),
+});
+
+/**
+ * Cognito User helper
+ */
+export const cognitoUser: RetryableMatcherHelper<
```

Comment:
We should start writing JS documentation. 
I started doing this [here](https://github.com/serverless-guru/sls-jest/pull/79/files#diff-9aeb505751d68c91640d7cf699dd4b4b66aed2d7eb5849dcf58009da87c0020fR42)

This should allow us to auto-generate doc later and it also gives more intellisense

![CleanShot 2024-01-18 at 14 02 08@2x](https://github.com/serverless-guru/sls-jest/assets/7089997/40dcdd6d-2285-45e7-9e82-bfd3ba7c1d8f)